### DISCUSSION TITLE
AEMET

### DISCUSSION THREAD

**Discussion by obaldius** [home-assistant/frontend#20501] [Configuration] [answered]
I have this entity from the AEMET integration

![image](https://github.com/home-assistant/frontend/assets/61756214/d3100815-2120-4e04-bf17-1a2d6763068a)

it features the forecast both daily or hourly.
however when I go to the entity itself under developer tools that info is not there

![image](https://github.com/home-assistant/frontend/assets/61756214/689cd79e-b41c-45fc-a705-2496ea7cb1b1)

how is that possible? how am I to use that forecast information? been looking for some other entity and nothing

**Comment by mbo18** [comment, marked as answer]
There is a service now to get this info. See weather.get_forecasts here: https://www.home-assistant.io/integrations/weather/
Check the forum for more info and help
